cp1404practicals/prac_05/emails.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404 Practical 05
+emails
+"""
+
+email = input("Email: ")
+email_book = {}
+while email != "":
+    temp_name = email.split("@")[0]  # splits email at "@" and takes the first part
+    temp_name = temp_name.split(".")
+    persons_name = ""
+    for name in temp_name:
+        persons_name += name.title() + " "
+    try:
+        name_change = input("Is your name {}? (Y/n) ".format(persons_name))
+        if name_change[0] in "Nn":
+            persons_name = input("Name: ")
+    except IndexError:
+        no = 1  # Awful placeholder as I'm out of ideas.
```

Comment:
Change this line to "pass" without the quotes. Maybe don't use exception handling next time unless you know you need it.

Reply:
I could only find "continue" and "break" to use there, cheers. Both your suggestions have been put in. Exception handling is needed to mimic the example outputs.